app/scripts/modules/huaweicloud/src/pipeline/stages/bake/HuaweiCloudBakeStageExecutionDetails.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { get } from 'lodash';
+
+import {
+  ExecutionDetailsSection,
+  IExecutionDetailsSectionProps,
+  StageFailureMessage,
+  SETTINGS,
+  IExecutionStage,
+} from '@spinnaker/core';
+
+function interpolate(text: string, stage: IExecutionStage) {
+  let newText = text;
+  const re = /{{([^{}]*)}}/;
+
+  while (-1 != newText.search(re)) {
+    newText = newText.replace(re, (substr: string) => {
+      return get(stage, substr.substring(2, substr.length - 2), undefined);
+    });
+  }
+
+  return newText;
+}
+
+export function HuaweiCloudBakeStageExecutionDetails(props: IExecutionDetailsSectionProps) {
+  const { stage, execution } = props;
+
+  return (
+    <ExecutionDetailsSection name={props.name} current={props.current}>
+      <div className="step-section-details">
+        <div className="row">
+          <div className="col-md-12">
+            <dl className="dl-horizontal">
```

Comment:
instead of: `<dl> <dt> <dd>`, prefer:
```js
<LabeledValueList>
  <LabeledValue label="Base OS" value={stage.context.baseOs} />
</LabeledValueList>
```